Lingua-App/Lingua/Lingua/Scenes/ProjectFormView/ProjectFormView.swift

```diff
@@ -49,46 +49,60 @@ struct ProjectFormView: View {
   }
 }
 
+// MARK: - Private View Builders
 private extension ProjectFormView {
-  var allFieldsValid: Bool {
-    let basicValidation = apiKeyValid && sheetIdValid && titleValid && outputPathValid
-    if project.type == .ios && project.swiftCodeEnabled {
-      return basicValidation && stringsDirectoryValid && outputSwiftCodeFileDirectoryValid
-    }
-    return basicValidation
-  }
-  
   @ViewBuilder
   func basicConfigurationFormSection() -> some View {
     Section(header: Text(Lingua.ProjectForm.configurationSection).font(.headline)) {
       Picker(Lingua.ProjectForm.platformPickerTitle, selection: $project.type) {
         ForEach(LocalizationPlatform.allCases) { type in
-          Text(type.title).tag(type)
+          Text(type.title)
+            .tag(type)
         }
-      }.padding([.leading], 8)
+      }.padding(.leading, 8)
       
-      ValidatingTextField(title: Lingua.ProjectForm.inputProjectName, validation: RequiredRule(), text: $project.title, isValid: $titleValid)
+      ValidatingTextField(
+        title: Lingua.ProjectForm.inputProjectName,
+        validation: RequiredRule(),
+        text: $project.title,
+        isValid: $titleValid
+      )
       
       InformationHolderView(content: {
-        ValidatingTextField(title: Lingua.ProjectForm.inputApiKey, validation: RequiredRule(), text: $project.apiKey, isValid: $apiKeyValid)
+        ValidatingTextField(
+          title: Lingua.ProjectForm.inputApiKey,
+          validation: RequiredRule(),
+          text: $project.apiKey,
+          isValid: $apiKeyValid
+        )
       }) {
-        Text(.init(Lingua.ProjectForm.apiKeyHelp)).padding()
+        Text(Lingua.ProjectForm.apiKeyHelp)
+          .padding()
       }
       
       InformationHolderView(content: {
-        ValidatingTextField(title: Lingua.ProjectForm.inputSheetId, validation: RequiredRule(), text: $project.sheetId, isValid: $sheetIdValid)
+        ValidatingTextField(
+          title: Lingua.ProjectForm.inputSheetId,
+          validation: RequiredRule(),
+          text: $project.sheetId,
+          isValid: $sheetIdValid
+        )
       }) {
-        Text(.init(Lingua.ProjectForm.sheetIdHelp)).padding()
```

Comment:
@borut-t The links are broken since with this initializer, SwiftUI handles localized strings differently, enabling certain features like link detection.